plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -244,21 +244,21 @@ private void deleteModelChunksAndController(
             bothDeleted.compareAndSet(true, b);
             if (countDownLatch.getCount() == 0) {
                 if (bothDeleted.get()) {
-                    log.debug("model chunks and model controller for model {} deleted successfully", modelId);
```

Comment:
Why don't we have model id there for non super admin users? The model id information can help debug and log analysis.

Reply:
whenever I get the model information, i re-used that info to detect if that model is hidden or not. But for delete model we don't perform any get request so I didn't introduce any extra api call.